templates/inspec/examples/google_spanner_instance/google_spanner_instance.erb

```diff
@@ -0,0 +1,11 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% spannerinstance = grab_attributes['spannerinstance'] -%>
+
+describe google_spanner_instance(project: <%= gcp_project_id -%>, name: <%= doc_generation ? "'#{spannerinstance['name']}'" : "spannerinstance['name']" -%>, config: <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>) do
+  it { should exist }
+  its('config') { should eq <%= doc_generation ? "'projects/#{gcp_project_id}/instanceConfigs/#{spannerinstance['config']}'" : "projects/#{gcp_project_id}/instanceConfigs/spannerinstance['config']" -%> }
```

Comment:
I'd prefer to make this something like:
`its('config') { should match <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>` }

Adding a string that gets interpolated within the test is tricky, especially because we generate both tests and docs from the same source

Reply:
ok. done